sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -772,7 +772,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
       Storage::Metadata Metadata;
       int64_t ContentLength = int64_t();
```

Comment:
Is this struct exposed to end user?

What was in `ContentLength` before this PR? I mean, when downloading partial file. Is this a breaking change?

Reply:
Whatever it is returned in the header.